packages/cli/src/services/import.service.ts

```diff
@@ -113,4 +396,32 @@ export class ImportService {
 			node.credentials[type] = nodeCredential;
 		}
 	}
+
+	async disableForeignKeyConstraints(transactionManager: EntityManager) {
```

Comment:
nit: why not having a single function called something like `toggleForeignKeyConstraints(enable: bool, transactionManager: EntityManager)` ?